### PR TITLE
Adding pyorerun

### PR DESCRIPTION
**Pull request by Ipuch**
Checklist

* [x]  Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
* [x]  License file is packaged
* [x]  Source is from official source.
* [x]  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
* [x]  If static libraries are linked in, the license of the static library is packaged.
* [x]  Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
* [x]  Build number is 0.
* [x]  A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe.
* [x]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
* [ ]  When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Ipuch** [comment]
I'm aware and I agree

**Comment by Ipuch** [comment]
@conda-forge/help-python

- I need help to make sure the automated conda-forge-linting service understand I consent for being a maintainer.
- I need guidance to understand why azure tests fail 

Thanks a lot !

File: recipes/pyorerun/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Step 1: Install dependencies from Conda Forge
+conda install -c conda-forge ezc3d trimesh rerun-sdk biorbd numpy -y
```
Comment (xhochy) [inline recipes/pyorerun/build.sh:4]: You should not be installing anything manually with conda here. All dependencies should be listed in the recipe.

This line can probably be removed without replacement

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/pyorerun/build.sh (hunk 1 above)
Comment (Ipuch) [reply recipes/pyorerun/build.sh]: Done

File: recipes/pyorerun/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 5526a7a60cf4a9a247165e763700745938eaceff8deaed507ecf967664d32641
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
```
Comment (xhochy) [inline recipes/pyorerun/meta.yaml:19]: You need to add your build system dependencies like hatchling here.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**